dmodel/dm-content.c

```diff
@@ -30,6 +30,8 @@ typedef struct {
   gchar *last_modified_date;
   gchar *license;
   gboolean featured;
+  gboolean can_print;
```

Comment:
The only improvement I can think of is collapsing these gboolean members into one bitfield, but no need to block the PR on that